lib/jekyll/tags/include.rb

```diff
@@ -163,7 +163,7 @@ def load_cached_partial(path, context)
       end
 
       def valid_include_file?(path, dir, safe)
-        !(outside_site_source?(path, dir, safe) || !File.exist?(path))
+        !(outside_site_source?(path, dir, safe) || !File.file?(path))
```

Comment:
I believe checking to see if it's a file is easier to read - affirmations are easier for us to grok than negations.